NGitLab/Models/Package.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Text.Json.Serialization;
+using NGitLab.Impl.Json;
+
+namespace NGitLab.Models
+{
+    public class Package
+    {
+        [JsonPropertyName("id")]
+        public int Id { get; set; }
+
+        [JsonPropertyName("package_id")]
+        public int PackageId { get; set; }
+
+        [JsonPropertyName("created_at")]
+        [JsonConverter(typeof(DateOnlyConverter))]
+        public DateTime CreatedAt { get; set; }
+
+        [JsonPropertyName("updated_at")]
+        [JsonConverter(typeof(DateOnlyConverter))]
+        public DateTime? UpdatedAt { get; set; }
+
+        [JsonPropertyName("size")]
+        public int Size { get; set; }
```

Comment:
File size is a long. I'm not sure GitLab would allow such big files, but you never know

```suggestion
        public long Size { get; set; }
```